conan/cache/conan_reference_layout.py

```diff
@@ -45,7 +53,7 @@ def export_sources(self):
         return os.path.join(self.base_folder, 'export_sources')
 
     def download_export(self):
-        return os.path.join(self.base_folder, "dl", "export")
+        return os.path.join(self.base_folder, "dl")
```

Comment:
No need to separate the export/pkg subfolders, they are in different base folders now.